### PR TITLE
chang button text

### PR DESCRIPTION
**Pull request by Mohammedal-abdulhamid**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class RootIndex extends React.Component {
         <Home />
         <Button
           onClick={handleClick}
-          text="Case Studies"
+          text="Our Success Stories"
```
Comment (pugoverflow) [inline src/pages/index.js]: Should be “see our success stories”

**Review by pugoverflow — CHANGES_REQUESTED** [review]

File: src/components/aboutUs.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 const AboutUs = () => (
-  <div className="m-2 bg-red-600 font-lato w-full p-4">
+  <div className="m-2 bg-red-600 font-lato  p-4">
```
Comment (pugoverflow) [inline src/components/aboutUs.js]: Remove extra space between classes, running linting before opening pull request will catch this!

**Review by Texx45 — CHANGES_REQUESTED** [review]
I think it should be 'See our success stories'

**Review by pugoverflow — APPROVED** [review]

**Comment by Mohammedal-abdulhamid** [comment]
changes done

**Review by Texx45 — APPROVED** [review]
Great job Mohammed

**Pull request closed without merging**